### ISSUE TITLE
Feature Request: Remove helped_code in json

### ISSUE THREAD

**Issue by leahcimto** [theforumhelpers/theforumhelpers.github.io#186]
**Is your feature request related to a problem?**
Not really. The helped_code thing in managers.json and curators.json is unneeded and can be removed.

**Describe alternatives you've considered**
None.

**Additional context**
Nothing should have to be changed in FHP.js.

**Comment by gosoccerboy5** [comment]
If I finish adding chiroyce and han614698 and then you guys merge this, will there be merge conflicts? Should I wait until this is possibly fixed?

**Comment by leahcimto** [comment]
There shouldn't be since they change different files.

**Comment by leahcimto** [comment]
Sorry, no there would be merge conflicts.

**Comment by gosoccerboy5** [comment]
> Sorry, no there would be merge conflicts. (I thought this was on the google analytics pr)

Do I wait until you guys finish fixing this, then clone that and add chiroyce/han(...) (to lazy to type 614698)? Or should I go ahead and add them, and then you can make a pr to fix this issue, based off of the edits I made? I feel like trying to simultaneously fix both issues would be a disaster because of potential conflicts

**Comment by leahcimto** [comment]
Since this isn't in progress yet you can make the add curators PR first.

**Comment by gosoccerboy5** [comment]
got it

**Comment by gosoccerboy5** [comment]
would this count as <a href="https://github.com/theforumhelpers/theforumhelpers.github.io/labels/type%3A%20dev-facing">(type: dev-facing)</a>?

**Comment by leahcimto** [comment]
Yes, thanks.

**Comment by gosoccerboy5** [comment]
My branch [remove-helpedcode](https://github.com/gosoccerboy5/theforumhelpers.github.io/tree/remove-helpedcode) may fix this

**Comment by Accio1** [comment]
@gosoccerboy5 Feel free to submit a PR. This will cut the size of those files which is always a good thing, especially if it isn't being used right now. Something similar might have to be added back if the "helped code" badges are added to the page, but that is not a priority right now.

**Comment by gosoccerboy5** [comment]
Ok. I can make a PR, but I think there might be some merge conflicts.. I'll see what I can do

**Comment by Accio1** [comment]
It's fine if there are merge conflicts. I can resolve them once you open a PR.

**Comment by Accio1** [comment]
Reopened because of #209

**Comment by leahcimto** [comment]
Looking over what was wrong with #206

**Comment by Steve0Greatness** [comment]
Is this meant to still be open?

**Comment by Accio1** [comment]
> Is this meant to still be open?

@Steve0Greatness Yes, the PR was merged but it broke the site so it was reverted and the issue was reopened.

**Comment by Steve0Greatness** [comment]
> > Is this meant to still be open?
> 
> @Steve0Greatness Yes, the PR was merged but it broke the site so it was reverted and the issue was reopened.

Makes sense.

**Comment by Accio1** [comment]
Aha, I found the issue with #206. After some bios, the comma wasn't removed , which caused it to not be read correctly. Removing these commas should fix the issue. @gosoccerboy5 are you still interested in this?

**Comment by Shluffy** [comment]
@Accio1 if they aren't I can do this
